cli/masternode/service.py

```diff
@@ -56,3 +57,33 @@ def close(self):
         return {
             'batch_id': close_masternode.batch_id,
         }, None
+
+    def set_bet(self, bet):
+        """
+        Set the masternode betting behaviour.
+
+        Arguments:
+            bet (string or integer, required): type of bet to set to the masternode account. Valid bet is
+                                               `min` or `max` as strings, or an integer value (e.g. 20).
+        """
+        if isinstance(bet, str):
+            bet = bet.upper()
+
+        if isinstance(bet, int):
+
+            if bet == 0:
```

Comment:
It validation should be in the bet field.

Reply:
No, it's just business logic.